File: frontend-android/clases/Producto.kt
package com.example.myapplication

data class Producto(
    var productoId: Int,
    var nombre: String,
    var descripcion: String,
    var puntos: Int
) {
    constructor() : this(
        productoId = 0,
        nombre = "",
        descripcion = "",
        puntos = 0
    )
}

File: frontend-wpf/C#Clases/Producto.cs
public class Producto
    {
        public int ProductoId { get; set; }
        public string Nombre { get; set; }
        public string Descripcion { get; set; }
        public int Puntos { get; set; }

        public Producto() { }

        public Producto(int productoId, string nombre, string descripcion, int puntos)
        {
            ProductoId = productoId;
            Nombre = nombre;
            Descripcion = descripcion;
            Puntos = puntos;
        }
    }